src/Caching/Hybrid/README.md

```diff
@@ -0,0 +1,3 @@
+# `HybridCache`
```

Comment:
It's a nice thought, I personally wouldn't leave this in though

Reply:
fine, nuked in 76d9143b81174b850b5212ab33f2a9af4ba9082e